### ISSUE TITLE
Audio/Video calls

### ISSUE THREAD

**Issue by MoMentalochka** [pocketnetteam/bastyon-chat#49]
- [ ] Init connection for server p2p

**Comment by ausov** [comment]
Do you plan to add calls to the chat app? I was thinking to adapt Element replacing matrix transport. Do you have any plans regarding this feature?

**Comment by Dennis-Volk** [comment]
Yes, we are already far along the curve. But we built a completely different front end, it is not the Element. We integrated into Bastyon, the open sourced decentralized social app and it is integrated with private key login.
